pkg/innerring/processors/container/process_container.go

```diff
@@ -256,7 +261,7 @@ func checkSubnet(subCli *morphsubnet.Client, cnr *containerSDK.Container) error
 	}
 
 	if !res.Allowed() {
-		return fmt.Errorf("user is not allowed to create containers in %s subnetwork", subID)
+		return fmt.Errorf("user is not allowed to create containers in %v subnetwork", subID)
```

Comment:
That was because during the work I forgot to provide `fmt.Stringer`. `%v` uses `fmt.Stringer` if it is provided, otherwise default format according to the type.